cmd/secondary-scheduler-operator/main.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
Current version of the operator-framework sdk proposes the main.go to be at the top of the root, deprecating the `cmd` folder to store main